test/project_types/rails/forms/create_test.rb

```diff
@@ -27,6 +27,14 @@ def test_title_can_be_provided_by_flag
         assert_equal("My New App", form.title)
       end
 
+      def test_aborts_if_title_includes_shopify
+        io = capture_io do
+          form = ask(title: "Shopify")
```

Comment:
Minor, but to make this a more 'realistic' test, I think it would be nice to use a title that contains "Shopify" but isn't exactly "Shopify", e.g. "Best Shopify Shipping App"

Reply:
Thank you for the feedback Andy!